core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -720,10 +720,9 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertThrows(classOf[InvalidGroupIdException], () => consumer1.commitSync())
   }
 
-  // Empty group ID only supported for classic group protocol
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
   @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly"))
-  def testConsumingWithEmptyGroupId(quorum: String, groupProtocol: String): Unit = {
+  def testConsumingWithoutGroupId(quorum: String, groupProtocol: String): Unit = {
```

Comment:
We can just remove this test case as it is duplicate to `testConsumingWithNullGroupId`

Reply:
Yes, removed it. Thanks.